subprojects/docs/src/samples/build-organization/composite-builds/basic/tests/sanityCheck.sample.conf

```diff
@@ -1,12 +1,12 @@
 commands: [{
     execution-subdirectory: my-app
     executable: gradle
-    args: help -q
+    args: tasks -q
```

Comment:
@mlopatkin Btw, does this change is same in terms of test coverage?
I mean that in both cases we covering only configuration phase.

Reply:
No, it isn't the same. The `help` only configures the `help` task (if the build author follows the recommendations and uses the lazy API, like `register`). This means that configuration blocks are not executed, and may allow errors to creep in, esp. in the case of Groovy scripts. The `tasks` task eagerly configure all other tasks, avoiding this problem. The updated plugin uses `tasks` as a sanity check target, but it cannot infer the structure of a composite build or a build that resides in a subfolder. For such snippets we have to override the generated sanity check test, and this is what I'm doing in this PR.

Reply:
Got it, thank you!